app/analysis/template.hbs

```diff
@@ -33,6 +34,9 @@
       {{#sort-header update=(action (mut ordering)) current=ordering by='task__project__name'}}Project{{/sort-header}}
       {{#sort-header update=(action (mut ordering)) current=ordering by='task__name'}}Task{{/sort-header}}
       {{#sort-header update=(action (mut ordering)) current=ordering by='comment'}}Comment{{/sort-header}}
+      <th>Needs review</th>
+      <th>Not billable</th>
+      <th>Verifier</th>
```

Comment:
Doesn't the backend support sorting those columns?

Reply:
No. It does not really make sense to sort by booleans here so we decided not to let them be sortable.

Reply:
Alright, thanks!

Reply:
As we changed Verified to Verifier it actually would make sense to be able to sort by verified_by. On second thought label might be better as Verified by.
I adjust the backend to allow this.